src/test/utils/testingHelper.ts

```diff
@@ -16,13 +16,14 @@ export type TestingRunParams = {
 
 type TestHelperParams = {
   testingRun: (testingRunParams: TestingRunParams) => void;
-} & { network: Network };
+} & { network: Network; onFork: boolean };
```

Comment:
```suggestion
} & { network: Network; onFork?: boolean };
```
I would suggest to make onFork optional, and it won't break our other tests use this function.